pkg/gui/networks_panel.go

```diff
@@ -0,0 +1,185 @@
+package gui
+
+import (
+	"strconv"
+
+	"github.com/fatih/color"
+	"github.com/jesseduffield/gocui"
+	"github.com/jesseduffield/lazydocker/pkg/commands"
+	"github.com/jesseduffield/lazydocker/pkg/config"
+	"github.com/jesseduffield/lazydocker/pkg/gui/panels"
+	"github.com/jesseduffield/lazydocker/pkg/gui/presentation"
+	"github.com/jesseduffield/lazydocker/pkg/gui/types"
+	"github.com/jesseduffield/lazydocker/pkg/tasks"
+	"github.com/jesseduffield/lazydocker/pkg/utils"
+	"github.com/samber/lo"
+)
+
+func (gui *Gui) getNetworksPanel() *panels.SideListPanel[*commands.Network] {
+	return &panels.SideListPanel[*commands.Network]{
+		ContextState: &panels.ContextState[*commands.Network]{
+			GetMainTabs: func() []panels.MainTab[*commands.Network] {
+				return []panels.MainTab[*commands.Network]{
+					{
+						Key:    "config",
+						Title:  gui.Tr.ConfigTitle,
+						Render: gui.renderNetworkConfig,
+					},
+				}
+			},
+			GetItemContextCacheKey: func(network *commands.Network) string {
+				return "networks-" + network.Name
+			},
+		},
+		ListPanel: panels.ListPanel[*commands.Network]{
+			List: panels.NewFilteredList[*commands.Network](),
+			View: gui.Views.Networks,
+		},
+		NoItemsMessage: gui.Tr.NoNetworks,
+		Gui:            gui.intoInterface(),
+		// we're sorting these networks based on whether they have labels defined,
+		// because those are the ones you typically care about.
+		// Within that, we also sort them alphabetically
+		Sort: func(a *commands.Network, b *commands.Network) bool {
```

Comment:
I assume we do something similar with another panel, but for networks is it actually true that we care more about labels than names? I suspect not

Reply:
Yes, it's exactly like volumes sorting, which fits to networks, i guess.

So, may be should we sort networks by driver at first and then by names?

Reply:
I'm thinking we just sort by name: I doubt the typical user has many networks configured anyway

Reply:
Done :)